format.sh

```diff
@@ -1,3 +1,13 @@
 #!/bin/sh
 
-find ./src -name '*.java' | xargs java -jar /app/google-java-format.jar -i
+set -e
+
+if [ $# -ne 0 ]; then
+    JAR_FILE="/app/google-java-format.jar"
+    if [[ -e $JAR_FILE ]]; then
+        find "$@" -name '*.java' | xargs java -jar /app/google-java-format.jar -i
```

Comment:
this condition will probably never be true. you're looking on the local filesystem here

Reply:
It is true when the script is run on the docker instance by `apigentools`